src/earn/EarnEnterAmount.tsx

```diff
@@ -0,0 +1,626 @@
+      networkId: token.networkId,
+    })
+    tokenAmount?.gt(token.balance)
+      ? addCryptoBottomSheetRef.current?.snapToIndex(0)
+      : reviewBottomSheetRef.current?.snapToIndex(0)
+  }
+
+  const onPressInfo = () => {
+    ValoraAnalytics.track(EarnEvents.earn_enter_amount_info_press)
+    infoBottomSheetRef.current?.snapToIndex(0)
+  }
+
+  return (
+    <SafeAreaView style={styles.safeAreaContainer}>
+      <CustomHeader style={{ paddingHorizontal: Spacing.Thick24 }} left={<BackButton />} />
+      <KeyboardAwareScrollView contentContainerStyle={styles.contentContainer}>
+        <View style={styles.inputContainer}>
+          <Text style={styles.title}>{t('sendEnterAmountScreen.title')}</Text>
+          <View style={styles.inputBox}>
+            <View style={styles.inputRow}>
+              <AmountInput
+                inputRef={tokenAmountInputRef}
+                inputValue={tokenAmountInput}
+                onInputChange={onTokenAmountInputChange}
+                inputStyle={styles.inputText}
+                autoFocus
+                placeholder={new BigNumber(0).toFormat(2)}
+                testID="SendEnterAmount/TokenAmountInput"
+              />
+              <View style={styles.tokenView} testID="SendEnterAmount/TokenSelect">
+                <>
+                  <TokenIcon token={token} size={IconSize.SMALL} />
+                  <Text style={styles.tokenName}>{token.symbol}</Text>
+                </>
+              </View>
+            </View>
+            <View style={styles.localAmountRow}>
+              <AmountInput
+                inputValue={token.priceUsd ? localAmountInput : '-'}
+                onInputChange={onLocalAmountInputChange}
+                inputRef={localAmountInputRef}
+                inputStyle={styles.localAmount}
+                placeholder={`${localCurrencySymbol}${new BigNumber(0).toFormat(2)}`}
+                testID="SendEnterAmount/LocalAmountInput"
+                editable={!!token.priceUsd}
+              />
+              {!token.balance.isZero() && (
+                <Touchable
+                  borderRadius={MAX_BORDER_RADIUS}
+                  onPress={onMaxAmountPress}
+                  style={styles.maxTouchable}
+                  testID="SendEnterAmount/Max"
+                >
+                  <Text style={styles.maxText}>{t('max')}</Text>
+                </Touchable>
+              )}
+            </View>
+          </View>
+        </View>
+        {showNotEnoughBalanceForGasWarning && (
+          <InLineNotification
+            variant={NotificationVariant.Warning}
+            title={t('sendEnterAmountScreen.notEnoughBalanceForGasWarning.title', {
+              feeTokenSymbol: prepareTransactionsResult.feeCurrencies[0].symbol,
+            })}
+            description={t('sendEnterAmountScreen.notEnoughBalanceForGasWarning.description', {
+              feeTokenSymbol: prepareTransactionsResult.feeCurrencies[0].symbol,
+            })}
```

Comment:
I guess this would just say "ETH" instead of "ETH on Arbitrum" which could be confusing, maybe we can have a follow up that adds network name to this